src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogTcpClient.cs

```diff
@@ -136,11 +142,45 @@ private void CloseConnection()
 #endif
             _stream = null;
             _client = null;
+            _connectionMatcher = null;
         }
 
         private bool IsConnectionClosed()
         {
-            return _client == null || _stream == null;
+            if (_client == null || _stream == null)
+            {
+                return true;
+            }
+
+#if NETSTANDARD1_3
```

Comment:
Do we need to match the other 1.x netstandard versions as well?

Reply:
It is not needed as other frameworks defined at https://github.com/DataDog/serilog-sinks-datadog-logs/blob/ac4ab777ccfac181efdfe74dc24f05ba4a0ed105/src/Serilog.Sinks.Datadog.Logs/Serilog.Sinks.Datadog.Logs.csproj#L4 implements `IPGlobalProperties`.